examples/SentimentAnalysis.java

```diff
@@ -0,0 +1,90 @@
+package ai.rev;
+
+import ai.rev.sentimentanalysis.models.SentimentAnalysisJobOptions;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisJob;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisJobStatus;
+import ai.rev.sentimentanalysis.models.SentimentAnalysisResult;
+import ai.rev.sentimentanalysis.SentimentAnalysisClient;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class SentimentAnalysis {
+
+  public static void main(String[] args) {
+    // Assign your access token to a String
+    String accessToken = "your_access_token";
```

Comment:
Use our conventional placeholder

```suggestion
    String accessToken = "<YOUR_ACCESS_TOKEN>";
```